Source/Library/Adapt.Analytics/ZerosAnalytic.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ******************************************************************************************************
+//  PassThrough.tsx - Gbtc
+//
+//  Copyright © 2021, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  08/02/2021 - C. Lackner
+//       Generated original version of source code.
+//  12/16/2021 - A. Hagemeyer
+//       Changed to a zeros analytic
+//
+// ******************************************************************************************************
+
+
+using Adapt.Models;
+using GemstoneCommon;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Adapt.DataSources
+{
+    /// <summary>
+    /// If value = 0 return NaN. Otherwise return the data
+    /// </summary>
+    
+    [AnalyticSection(AnalyticSection.DataCleanup)]
+    [Description("Zeros: Returns NaN if the value was 0.")]
```

Comment:
Descriptions should be more generic. `NaN` just implies no signal and will be ignored by the UI. So for the user this is not helpfull. Maybe `Removes zeros from the signal` would be a better description.